pkg/webui/lib/shared-messages.js

```diff
@@ -87,7 +87,9 @@ export default defineMessages({
   collaboratorEditRights: 'Edit rights of {collaboratorId}',
   collaboratorId: 'Collaborator ID',
   collaboratorIdPlaceholder: 'collaborator-id',
+  collaboratorWarningSelf: 'Changing your own rights could result in loss of access',
   collaboratorModalWarning: 'Are you sure you want to remove {collaboratorId} as a collaborator?',
+  collaboratorModalWarningSelf: 'Are you sure you want to remove yourself as a collaborator?',
```

Comment:
```suggestion
  collaboratorModalWarningSelf: 'Are you sure you want to remove yourself as a collaborator? Access to this entity will be lost until it is granted again by a different collaborator.',
```

Reply:
Reworded it a bit:

- Used "someone else" instead of "a different collaborator" because it could also be an admin
- Used "adds you as collaborator again" instead of "it is granted again"

<img width="655" alt="Screen Shot 2021-06-02 at 16 01 17" src="https://user-images.githubusercontent.com/181308/120494077-c963e580-c3bb-11eb-8db7-15e93b94f5d4.png">